_data/users.yml

```
@@ -407,6 +415,14 @@
   urltext:
   url: https://shopify.engineering/topics/data-science-engineering
   # https://shopify.engineering/faster-trino-query-execution-infrastructure
+- name: SK Telecom
+  anchor: sk_telecom
+  logo: /assets/images/logos/sk-telecom.png
+  logosmall: /assets/images/logos/sk-telecom-small.png
+  testimonial: |
+    TBD
+  urltext:
+  url: https://www.sktelecom.com/
```

```suggestion
  url: https://www.sktelecom.com/index_en.html
```
How about linking eng version